srcpkgs/niri/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'niri'
+pkgname=niri
+version=0.1.9
+revision=1
+build_style=cargo
+configure_args="--no-default-features --features xdp-gnome-screencast"
+hostmakedepends="pkg-config"
+makedepends="eudev-libudev-devel libxkbcommon-devel libinput-devel libdisplay-info-devel pipewire-devel pango-devel libseat-devel clang libgbm-devel"
+depends="eudev-libudev libxkbcommon libinput libpipewire pango libseat libgbm libdisplay-info"
+short_desc="Scrollable-tiling Wayland compositor"
+maintainer="joetroll <joetroll@proton.me>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/YaLTeR/niri"
+changelog="https://github.com/YaLTeR/niri/releases"
+distfiles="https://github.com/YaLTeR/niri/archive/refs/tags/v${version}.tar.gz"
+checksum=70427efcde50a9d262c341befbd4df4e24557afd8311753bf912a4be1bf243a7
+
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	export RUSTFLAGS='-C target-feature=+crt-static'
```

Comment:
why?

Reply:
i've opened an issue about this, it just fails to find the libraries.

Reply:
works fine with these changes (basically just applying what i said above):
```diff
diff --git a/srcpkgs/niri/template b/srcpkgs/niri/template
index 9af1ff19698..1c15b027e85 100644
--- a/srcpkgs/niri/template
+++ b/srcpkgs/niri/template
@@ -4,9 +4,9 @@ version=0.1.9
 revision=1
 build_style=cargo
 configure_args="--no-default-features --features xdp-gnome-screencast"
-hostmakedepends="pkg-config"
-makedepends="eudev-libudev-devel libxkbcommon-devel libinput-devel libdisplay-info-devel pipewire-devel pango-devel libseat-devel clang libgbm-devel"
-depends="eudev-libudev libxkbcommon libinput libpipewire pango libseat libgbm libdisplay-info"
+hostmakedepends="pkg-config clang18-devel"
+makedepends="eudev-libudev-devel libxkbcommon-devel libinput-devel libgbm-devel
+ libdisplay-info-devel pipewire-devel pango-devel libseat-devel clang18-devel"
 short_desc="Scrollable-tiling Wayland compositor"
 maintainer="joetroll <joetroll@proton.me>"
 license="GPL-3.0-or-later"
@@ -15,10 +15,6 @@ changelog="https://github.com/YaLTeR/niri/releases"
 distfiles="https://github.com/YaLTeR/niri/archive/refs/tags/v${version}.tar.gz"
 checksum=70427efcde50a9d262c341befbd4df4e24557afd8311753bf912a4be1bf243a7
 
-if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
-	export RUSTFLAGS='-C target-feature=+crt-static'
-fi
-
 post_install() {
 	vmkdir usr/share/wayland-sessions
 	vmkdir usr/share/xdg-desktop-portal

```

this also fixes cross.